roles/ceph-defaults/tasks/check_socket.yml

```diff
@@ -0,0 +1,21 @@
+---
+# These checks are used to avoid running handlers at initial deployment.
+- name: check for a ceph socket
+  shell: |
+    {{ docker_exec_cmd }} bash -c 'stat {{ rbd_client_admin_socket_path }}/*.asok > /dev/null 2>&1'
```

Comment:
@andrewschoen yes, `docker_exec_cmd` has a default value set to `''` then it's set [here](https://github.com/ceph/ceph-ansible/blob/refact/roles/ceph-defaults/tasks/facts.yml#L74-L77) so we can use this check for both `ceph-common` and `ceph-docker-common`, at least this was already used that way before that PR, I just moved all this from `ceph-mon` to `ceph-defaults`